developer-tools.md

```diff
@@ -494,6 +494,27 @@ in the Eclipse install directory. Increase the following setting as needed:
 256M
 ```
 
+<h4>IDE with sbt</h4>
+
+When you use IDEs like VSCode or Cursor and you build from terminal with `sbt`,
```

Comment:
Can we also provide tips for step-by-step setups for these IDEs? In the `IDE setup` section, we only have guidelines for `IntelliJ`. It's better to have instructions for them too